hs/src/Reach/Simulator/Core.hs

```diff
@@ -793,7 +788,7 @@ bindConsensusMeta (DLRecv {..}) actorId accId = do
 
 instance Interp LLProg where
   interp (LLProg _at _llo slparts _dli _dex _dvs _apis _evts step) = do
-    registerParts $ M.keys $ sps_ies slparts
+    registerParts $ M.toList $ sps_ies slparts
```

Comment:
I try to always use `toAscList` because I am nervous about depending on the order and it changing someday